guide/contributing/developer-guide.md

```diff
@@ -133,11 +169,55 @@ If this does not resolve your issue, there could be an issue with duplicate depe
 To solve this, we use [lerna-update-wizard](https://www.npmjs.com/package/lerna-update-wizard) using the command `npx lernaupdate --dedupe` to
 assure duplicate packages are on the same version.
 
-## Running and updating tests
+## Quality checklist
+
+Be sure to fulfill all items in this quality checklist for TDS components:
+
+- [Accessibility](#accessibility): is the component accessible, or makes direct suggestions to building accessible experiences?
+- [Unit tests](#unit-tests): is the component's functionality and rendering tested?
+- [e2e tests](#e2e-tests): is there an accurate baseline image?
+- Documentation: is the component documented with examples? Are there usage criteria for designers, and accessibility considerations clearly written?
+- [npm package](#npm-package): is the package manifest appropriately set up? Does it build? Is the distributable build lean?
+- [Manual tests](#manual-tests): has the component been tested in [supported browsers][faq-browsers]?
+  - **Mobile first**: does it work properly in mobile browsers, on mobile devices? Is the component responsive?
+- User story: are acceptance criteria fulfilled, and coded component matches designs?
+
+The following sections go into greater detail on how to fulfill the component quality checklist.
+
+### Accessibility
+
+Accessibility mindfulness is essential when building design system components. After making considerations such as:
```

Comment:
"After making considerations such as" sounds awkward with no follow-up after it.